paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -294,6 +294,7 @@ void CpuPassStrategy::EnableMKLDNN() {
              // Disabled due to topology-dependent speed-up
              //  "fc_mkldnn_pass",
              //  "fc_act_mkldnn_fuse_pass",
```

Comment:
Verify if we can turn these passes on, please.

Reply:
@Silv3S Hi, could you please add a VIT/OCR test in the API test. And use insert_pass, to enable the fc_mkldnn_pass and fc_act_mkldnn_fuse_pass? I will find you reference

Reply:
@Silv3S  You may need to count together with passes inside `CpuPassStrategy()`, because those are pushed firstly into `pass_` and then mkldnn passes followed. 

You can verify if it is correct by running and observing the execution log.

The fc related two passes are currently commented now, because in some models with the fc passes, fp32 oneDNN is slower than fp32 original. We need to figure out a good way later.